src/components/Map/Map.jsx

```diff
@@ -24,7 +24,12 @@ export async function getNewMarkers(categories) {
     );
     const filtersUrlQueryString = allCheckboxes.filter(n => n).join('&');
     const locations = await httpService.getLocations(filtersUrlQueryString);
-    return locations.map(location => <MarkerPopup place={location} key={location.metadata.UUID} />);
+
+    let markers = locations.map(location => {
+        const locationKey = window.USE_LAZY_LOADING ? location.UUID : location.metadata.UUID;
+        return <MarkerPopup place={location} key={locationKey} />;
```

Comment:
The app crashes and I suspect that here might be the problem.
It crashes when `USE_LAZY_LOADING` is set to false or is not present in the config. In that case you pass simply `{UUID, position}` as the `place` argument to `MarkerPopup`.

Reply:
Should work now. Tough place which needed a change wasn't here :)